python/lsst/afw/display/interface.py

```diff
@@ -523,27 +526,26 @@ def pan(self, colc=None, rowc=None, origin=afwImage.PARENT):
         """!Pan to (rowc, colc); see also zoom"""
 
         self.zoom(None, colc, rowc, origin)
-
+    
     def interact(self):
         """!Enter an interactive loop, listening for key presses in display and firing callbacks.
-
-        Exit with q, \c CR, or \c ESC
-    """
-
-        while True:
+            Exit with q, \c CR, or \c ESC
+        """
+        interact_finished = False
```

Comment:
[Coding standard](https://developer.lsst.io/coding/python_style_guide.html#naming-conventions) prefers `interactFinished`, ugly though it is.